code/datums/shuttles.dm

```diff
@@ -312,6 +319,22 @@
 	credit_cost = 10000
 	danger_level = SHUTTLE_DANGER_SUBPAR
 
+/datum/map_template/shuttle/emergency/funnypod
+	suffix = "funnypod"
+	name = "Comically Large Escape Pod"
+	description = "A bunch of scrapped escape pods glued together."
+	admin_notes = "This shuttle will 100% cause mayhem, as the space avaiable is 1x23 and anyone can open the door in the end."
+	credit_cost = 2000
+	danger_level = SHUTTLE_DANGER_SUBPAR
+
+/datum/map_template/shuttle/emergency/honkco
+	suffix = "honkco"
+	name = "Honk.Co shuttle"
+	description = "From the creators of Snappop(tm)!, the signature Honk.Co shuttle is now avaiable to purchase, with no usable chairs and filled with bananas, clown artifacts and all types of clowns."
+	admin_notes = "Is what you expect. Bananium shuttle full of clowns, 4 staffs of the honk mother, bananas everywhere and cluwnes."
```

Comment:
No, its not "what you expect."

All these clowns inside the ship are coded as *hostile* mobs and could get people killed.

Also, the majority of the ship is made from a resource I could see players stripping and using to build stuff, rather than actually enjoying the shuttle. Something to keep in mind.

Reply:
Clowns aren't hostile on sight, they retaliate

Reply:
![imagen](https://github.com/BeeStation/BeeStation-Hornet/assets/148357670/7c0bf4a0-89ad-46a4-ac7c-a393b79167a7)